src/adapters/recommended_adapter.js

```diff
@@ -7,7 +7,17 @@ class RecommendedAdapter {
   constructor () {
     this.results = new RecommendedResults();
     webChannel.on('printable-key', debounce((data) => {
-      this.search(data.query);
+
+      // Trim whitespace before performing search.
+      let query = data.query.replace(/^\s+/, '').replace(/\s+$/, '');
```

Comment:
I originally had a more complex regex, but ended up stripping it down to this, which is functionally close to `.trim()`. Good catch.